api_tests/nodes/views/test_node_provider_file_metadata_create.py

```diff
@@ -0,0 +1,350 @@
+from __future__ import unicode_literals
+
+import pytest
+
+from api.base.settings.defaults import API_BASE
+from api_tests import utils as api_utils
+from framework.auth.core import Auth
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    UserFactory,
+)
+from osf.models import QuickFilesNode
+from website.util import permissions
+
+
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
+
+
+@pytest.mark.django_db
+class TestNodeProviderFileMetadataCreateView:
+
+    @pytest.fixture()
+    def node(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def node2(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def file(self, user, node):
```

Comment:
very, very minor, especially since I know this came from elsewhere: `file` is a `type` in python.

Reply:
`file_one`